src/test/java/org/takes/facets/fork/TkMethodsTest.java

```diff
@@ -62,4 +67,19 @@ public void throwsExceptionOnActinOnUnproperMethod() throws
             new RqFake(RqMethod.GET)
         );
     }
+
+    /**
+     * TkMethods can return 405 status when acting on unknown method method.
+     * @throws IOException if any I/O error occurs.
```

Comment:
@victornoel descriptions in javadoc for throws/params/return should start with capital letter, and should not end with a dot.